docs/canarytokens/actions.md

```diff
@@ -431,9 +445,9 @@ print(r.json())
 
 </APIDetails>
 
-## Fetch all Canarytokens
+## Paginate Canarytokens
 
-<APIDetails :endpoint="$page.frontmatter.endpoints.fetch_canarytokens">
+<APIDetails :endpoint="$page.frontmatter.endpoints.paginate">
```

Comment:
We will need to update the example request and response as well. It'll be at least lines 446 for the `curl` example, 457 for the `python` example, and 473 onwards for the reponse.

Reply:
Thanks, missed that when copying the other one over, should be sorted now.